### DISCUSSION TITLE
URGENT - store is completely down

### DISCUSSION THREAD

**Discussion by jezzzm** [swellstores/community#117] [Help] [answered]
All we are seeing is this. Graphql api is down. REST api seems to be working. We use graphql for auth so all our users have been kicked out.

![image](https://user-images.githubusercontent.com/26472490/215241932-daab6aa4-fb9c-4393-8467-d7a92356bbf3.png)

cannot access admin ui to reach out to support - I'm sorry to send this here

**Comment by steven-k1** [comment, marked as answer]
Hello @jezzzm

We are aware of the issue and are actively looking into it.

For any updates, please follow our status page here: https://status.swell.store/

**Reply by jezzzm** [reply]
hi @StevenSwell , thanks - this is not limited to our store right?

**Reply by steven-k1** [reply]
@jezzzm No, the issue is wider than just your store.

However, we are seeing services stabilize and will continue to monitor them.

Would you be able to confirm this from your end?

**Reply by jezzzm** [reply]
yes we are back in action. thanks for your swift work!

**Reply by jezzzm** [reply]
We are back down @StevenSwell ... this time apache welcome page
![image](https://user-images.githubusercontent.com/26472490/215268097-a3f15387-0aa4-49c7-a6d9-918bbfa1f166.png)

**Reply by jezzzm** [reply]
annnnd back up
